client-app/src/pages/account/views/AccountOrders.vue

```diff
@@ -1,12 +1,88 @@
 <template>
   <div>
-    Orders Page
+    <loading :active.sync="isLoading"></loading>
+
+    <b-table id="orders-list"
+             striped
+             hover
+             :items="ordersList.orders"
+             :fields="ordersList.listConfig.columns">
+      <!-- A custom formatted header cell for field 'name' -->
+      <template v-slot:head()="data">
+        <span class="text-info">{{ $t( `grids.orders_list.columns.${data.column.split('.').join('_')}`) }}</span>
+      </template>
+
+      <!-- A custom formatted column -->
+      <template v-slot:cell(createdDate)="data">
```

Comment:
for display money better also define some components for such cases where API won't return formatted data. But this can be postponed on the future.

Reply:
I agree. In a feature.